osbs/api.py

```diff
@@ -536,17 +536,23 @@ def create_worker_build(self, *args, **kwargs):
         modifications:
             - platform param is required
             - release param is required
-            - inner template set to worker_inner.json if not set
+            - arrangement_version param is required, which is used to
+              select which worker_inner:n.json template to use, and
+              removed from params
+            - inner template set to worker_inner:version.json if not set
             - outer template set to worker.json if not set
             - customize configuration set to worker_customize.json if not set
 
         :return: BuildResponse instance
+
         """
-        for required in ('platform', 'release'):
-            if not kwargs.get(required):
+        for required in ('platform', 'release', 'arrangement_version'):
+            if kwargs.get(required) is None:
```

Comment:
I've reimplemented this differently now. Rather than remember whether each param must be truthy, let's start arrangement_version at 1 so they all must be truthy.